consensus/core/src/dag_state.rs

```diff
@@ -202,6 +202,48 @@ impl DagState {
             .collect()
     }
 
+    pub(crate) fn contains_block_in_cache_or_store(
```

Comment:
Just `contains_block()` and `contains_blocks()`? Cache or store is an implementation detail.